src/test/java/org/opentripplanner/middleware/utils/TemplateUtilsTest.java

```diff
@@ -50,7 +60,7 @@ public static List<TemplateRenderingTestCase> createTemplateRenderingTestCases()
         // Trip Monitor Notifications tests (for CheckMonitoredTrip#sendNotifications).
         Map<String, Object> notificationsData = Map.of(
             "tripId", "18f642d5-f7a8-475a-9469-800129e6c0b3",
-            "notifications", List.of("Test notification.", "Another notification.")
+            "notifications", List.of(new TripMonitorNotification(NotificationType.INITIAL_REMINDER, "This is the initial reminder text"), new TripMonitorNotification(NotificationType.DEPARTURE_DELAY, "This is the departure delay text"))
```

Comment:
Reformat placing each `new TripMonitorNotification(...)` on a new line with the correct indent.